syntax = "proto3";

package server2client;

message msg_content
{
	int32 type = 1;
	bytes datas = 2;
}

message msg_connect
{
	enum type_val{
		id = 0;
	}
	repeated int32 arr_ints = 1;
	repeated string arr_strings = 2;
}

message msg_testmap
{
	enum type_val{
		invalid = 0;
		id = 5;
	}
	map<int32, string> map_info = 1;
}